crates/dbsp/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 #![cfg_attr(docsrs, feature(doc_cfg))]
-#![doc = include_str!("../../../README.md")]
+#![doc = include_str!("../README.md")]
```

Comment:
I am afraid the top-level readme doesn't apply to this crate anymore. Your comments in `circuit_builder.rs` are probably a good foundation of a readme for this crate. We just need to preface it with a one-paragraph summary and a link to the paper and the tutorial.

Reply:
I wrote some proper crate documentation for what I just pushed.